examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/Pages/Index.cshtml.cs

```diff
@@ -1,13 +1,12 @@
 ﻿namespace WebDemoWithEventHub.Pages
 {
     using Microsoft.AspNetCore.Mvc.RazorPages;
-    using Microsoft.Extensions.Options;
 
     public class IndexModel : PageModel
     {
-        public IndexModel(IOptionsSnapshot<Settings> options)
+        public IndexModel(ISettingsProvider settingsProvider)
```

Comment:
We only change how the application detects for config changes (from poll model to push model). This shouldn't change how the application consumes configuration. I don't expect this file changes at all.

Reply:
Undid this change.

---
In reply to: [426103473](https://github.com/Azure/AppConfiguration/pull/329#discussion_r426103473) [](ancestors = 426103473)